docs/multiple-ako.md

```diff
@@ -0,0 +1,69 @@
+# Multiple AKO instances in a cluster
+
+This feature allows to run multiple instances of AKO per cluster.
+
+## Overview
+
+Prior to AKO 1.7.1, single instance of AKO was responsible to process kubernetes/openshift objects updates and creates corresponding AVI objects at controller.
+
+With AKO 1.7.1, multiple AKO instance can be deployed in a cluster to create namespace based isolation. This will help to club group of kubernetes/openshift namespaces under one AKO. So that AKO will handle all kubernetes/openshift events for those namespaces only. To run multiple AKO, following two features will be used.
```

Comment:
Nit: multiple AKO **instances**

suggesting ~This will help to club group of kubernetes/openshift namespaces under one AKO...~
This will allow AKO to operate over a group of kubernetes namespaces, in order to handle objects from these namespaces only